src/card.css

```diff
@@ -0,0 +1,101 @@
+:root {
+  --main-color: #060b35;
+  --secondary-color: #616070;
+  --button-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  align-self: center;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-color);
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary-color);
+  font-weight: 400;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-top: 16px;
+}
+
+.card__rating-counter {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: var(--main-color);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 24px;
+}
+
+.card__price-label {
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+  color: var(--secondary-color);
+}
+
+.card__price-price {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+  color: var(--main-color);
+}
+
+.card__purcache-button {
+  margin-top: 16px;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  color: #fff;
+  text-transform: uppercase;
+  text-decoration: none;
```

Comment:
Organize your styles
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)